lib-utilities/services/rpcService.go

```diff
@@ -78,3 +85,124 @@ func InitializeService(serviceName string) error {
 	return nil
 
 }
+
+// gRPC implementation starts here. TODO: remove this comment after the removal of go micro implementation.
+
+type serviceType int
```

Comment:
What values can this assume ? Can this be an enum ?

Reply:
yes @Bharath-KKB this is an enum, and the values for this enum are `serverService` & `clientService` mentioned in line 94 and 95